rdt-app/src/main/java/io/ona/rdt_app/activity/RDTJsonFormActivity.java

```diff
@@ -95,4 +95,10 @@ public String getRdtType() {
     public void setRdtType(String rdtType) {
         this.rdtType = rdtType;
     }
+
+    @Override
+    public void onStop() {
+        setRdtType(ONA_RDT);
```

Comment:
Why are we setting the type to ONA_RDT onStop()? :thinking:

Reply:
There was a bug I was testing out a fix for.

Turns out it was in the UW code so this is not really necessary.

Removed.